1137-N-th-Tribonacci-Number.py

```diff
@@ -0,0 +1,45 @@
+'''
+The Tribonacci sequence Tn is defined as follows: 
+
+T0 = 0, T1 = 1, T2 = 1, and Tn+3 = Tn + Tn+1 + Tn+2 for n >= 0.
+
+Given n, return the value of Tn.
+
+ 
+
+Example 1:
+
+Input: n = 4
+Output: 4
+Explanation:
+T_3 = 0 + 1 + 1 = 2
+T_4 = 1 + 1 + 2 = 4
+
+Example 2:
+
+Input: n = 25
+Output: 1389537
+
+ 
+
+Constraints:
+
+    0 <= n <= 37
+    The answer is guaranteed to fit within a 32-bit integer, ie. answer <= 2^31 - 1.
+
+'''
+class Solution:
+    def tribonacci(self, n: int) -> int:
+        a = 0
+        b = c = 1
+        if n == 0:
+            return 0
+        elif n == 1 or n == 2:
+            return 1
+        else:
```

Comment:
This is not required

Reply:
But won't it make a problem when the value of n is less than 3, as I have initialized i as 3? or will using the for loop starting from 2 manage that as well?

Reply:
If the value is less than 3 then it will never come to this position as you already have return statements for all those cases